test/screenshot/spec/mdc-elevation/classes/baseline.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<!--
+  Copyright 2018 Google Inc. All Rights Reserved.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+      https://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+-->
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <title>Baseline Elevation - MDC Web Screenshot Test</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="../../../out/mdc.elevation.css">
+    <link rel="stylesheet" href="../../../out/mdc.typography.css">
+    <link rel="stylesheet" href="../../../out/spec/fixture.css">
+    <link rel="stylesheet" href="../../../out/spec/mdc-elevation/fixture.css">
+  </head>
+
+  <body class="test-container mdc-typography">
+    <main class="test-viewport test-viewport--mobile">
+      <div class="test-layout">
+        <div class="test-cell test-cell--elevation"><span class="mdc-elevation--z0">0</span></div>
```

Comment:
Do you think it might be a good idea to apply a class to each span that gives it a fixed size? e.g. in case of text rendering flakes between browsers/runs...

Reply:
`test-cell--elevation` already sets a fixed width/height for that exact reason 😁

Reply:
But that's the cell, not the thing being elevated.

My thought was that if the text causes flakes, that will cause flakes on the box shadows as well.

Reply:
Ahh, I see what you're saying. Good idea - I'll do that!

Reply:
Done